media-libs/netpbm/netpbm-10.86.21.ebuild

```diff
@@ -0,0 +1,208 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="A set of utilities for converting to/from the netpbm (and related) formats"
+HOMEPAGE="http://netpbm.sourceforge.net/"
+SRC_URI="https://github.com/ceamac/netpbm-make-dist/releases/download/v10.86.21/${P}.tar.xz"
```

Comment:
i don't think we want to do this.  we'll just maintain the archives in Gentoo mirrors.

Reply:
Agreed it sucks, but infra disabled dumping tarballs into mirror://gentoo, so the only choice is this or devspace which are equivalent

Reply:
https://devmanual.gentoo.org/general-concepts/mirrors/#suitable-download-hosts

mirror://gentoo is deprecated and I don't have a dev space, so this was the closest I could do.  At least it's on github.